src/components/Modal/components/Dialog/index.ts

```diff
@@ -1,4 +1 @@
-import Dialog from './Dialog';
```

Comment:
Can you please revert these stylistic changes for the moment?

I agree that the `export {default, Props} from './Dialog';` style is better but  these changes adds a some noise when comparing the build.

Ideally I'd like to be able to compare the build output and see that they are identical, and this adds changes.